kernel/transaction/type/xa/core/pom.xml

```diff
@@ -53,6 +53,11 @@
             <groupId>com.mysql</groupId>
             <artifactId>mysql-connector-j</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.oracle.database.jdbc</groupId>
+            <artifactId>ojdbc8</artifactId>
+            <version>23.6.0.24.10</version>
```

Comment:
- Please don't do this, it's not convenient to maintain the version number. You should add `<ojdbc8.version>23.6.0.24.10</ojdbc8.version>` in the top level `pom.xml`, and then use `${ojdbc8.version}` in this place.